build_config/lib/src/builder_definition.dart

```diff
@@ -55,7 +55,7 @@ class BuilderDefinition {
   /// The name of the dart_library target that contains `import`.
   ///
   /// May be null or unreliable and should not be used.
-  @deprecated
+  @Deprecated('Do not use')
```

Comment:
Uh...could we be more clear why? 😄

Reply:
It's not very user facing and the doc comment says it's unreliable.  I don't think we need to be super specific with these.